controllers/apartments_controller.rb

```diff
@@ -0,0 +1,40 @@
+# List all apartments (hardcode into the erb file)
+get "/apartments" do
```

Comment:
:+1:  looks great, see if you can include delete and update functionality as well